downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile.java

```diff
@@ -0,0 +1,413 @@
+package org.reactome.release.downloadDirectory;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.model.ClassAttributeFollowingInstruction;
+import org.gk.model.GKInstance;
+import org.gk.model.InstanceUtilities;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.SchemaClass;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.*;
+
+public class GenerateGOAnnotationFile {
+
+    private static final Logger logger = LogManager.getLogger();
+    private static final String CELLULAR_COMPONENT = "Cellular Component";
+    private static final String MOLECULAR_FUNCTION = "Molecular Function";
+    private static final String BIOLOGICAL_PROCESS = "Biological Process";
+    private static final List<String> goTerms = new ArrayList<>(Arrays.asList(CELLULAR_COMPONENT, MOLECULAR_FUNCTION, BIOLOGICAL_PROCESS));
+    private static final String GOA_FILENAME = "gene_association.reactome";
+    private static final int MAX_RECURSION_LEVEL = 2;
+    private static final String uniprotDbString = "UniProtKB";
+    private static final String PROTEIN_BINDING_ANNOTATION = "0005515";
+    private static final List<String> speciesWithAlternateGOCompartment = new ArrayList<>(Arrays.asList("11676", "211044", "1491", "1392"));
+    private static final List<String> microbialSpeciesToExclude = new ArrayList<>(Arrays.asList("813", "562", "491", "90371", "1280", "5811"));
```

Comment:
I think it's ok keeping them all contained here, since this is the only place they are being used. If they start getting used in other places, then a separate file would make sense.

Reply:
Fair enough.  Could they just be separated into a group with a space after the logger and before `goaLines` so it's clear they are a distinct group?